main.tex

```diff
@@ -770,16 +773,19 @@ \subsubsection*{Provenance entities}
 Ideally, it can be linked to a complete experiment specification (e.g., as a SED-ML~\cite{Koehn2008} or SESSL~\cite{Ewald2014} file or simply as the execution code in a general purpose programming language) and to documentation in a standard format that applies reporting guidelines such as MIASE for simulation experiments~\cite{Waltemath2011}.
 Diverse simulation experiments might be applied to serve the analysis, calibration, and validation of a simulation model.
 
-To further structure the set of simulation experiments applied, we distinguish simulation experiments whether they are used for optimization, sensitivity analysis, perturbation, parameter scan, steady-state analysis, or time course analysis.
+To further structure the set of applied simulation experiments, we distinguish simulation experiments by whether they are used for optimization, sensitivity analysis, perturbation, parameter scan, steady-state analysis, or time course analysis.
 This list is neither complete nor are the categories disjoint, and, given a different set of simulation studies, they will likely be subject to renaming, extension, and refinement.
-We have identified optimization experiments to be all experiments where an implicit or explicit objective function is being used.
+
+We have defined optimization experiments to be all experiments where an implicit or explicit objective function is used.
 This includes parameter estimation as well as manual parameter fitting experiments.
 If these succeed, the \CSM{} activity will produce a new (calibrated) \SM{}.
-In a sensitivity analysis, more than one parameter value is being changed and some kind of sensitivity coefficient is being calculated.
-As perturbations, we have interpreted experiments where the value of one (or more) parameter is being changed to another (just one) value, for example, to mimic a knock-out experiment.
+In a sensitivity analysis, more than one parameter value is changed and some kind of sensitivity coefficient is calculated.
+As perturbations, we have interpreted experiments where the value of one (or more) parameter is changed to another (just one) value, for example, to mimic a knock-out experiment.
```

Comment:
Not exactly sure what "As perturbations" means here? This could definitely be due to my limited knowledge in the field

Reply:
Perturbation is a category for simulation experiments we are using. Do you have a good idea for rephrasing it?

Reply:
Maybe "For perturbations,"? The "As " is what I think might be an issue

Reply:
Yes I like it!